themes/default/partials/search/helpers-js.hbs

```diff
@@ -2,45 +2,35 @@
 {{> common-helpers-js }}
 
 <script type="text/javascript">
+"use strict";
 
 if (!window.helpers) {
-  window.helpers = {}
+  window.helpers = {};
 }
 
-window.helpers.searchFiles = (searchModel, files) => {
+window.helpers.searchFiles = function (searchModel, files) {
   if (searchModel !== '') {
-    let searchedFiles = files.filter((file) => {
-      let fileContent = JSON.stringify(file).toLowerCase()
-      let searchParam = searchModel.toLowerCase()
-      return fileContent.includes(searchParam)
-    })
-
-    return searchedFiles
+    var searchedFiles = files.filter(function (file) {
+      var fileContent = JSON.stringify(file).toLowerCase();
+      var searchParam = searchModel.toLowerCase();
+      return fileContent.includes(searchParam);
```

Comment:
Did `includes` cause any issues? Looks like it is not supported in ie :( 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes

Reply:
Looks like we polyfill this upstream from here. LGTM